components/infra-proxy-service/migrations/migrations.go

```diff
@@ -331,3 +313,12 @@ func (s *MigrationServer) ConfirmPreview(ctx context.Context, req *request.Confi
 		MigrationId: req.MigrationId,
 	}, nil
 }
+
+func StreamErr(err error, ctx context.Context, stream service.MigrationDataService_UploadFileServer, migServer *MigrationServer, migrationId, serverId, errMsg string) {
+	log.Errorf(errMsg)
```

Comment:
Can you print the error message properly, like "Unable to upload the zip file %s',err.Errors()) and not take the string in the function itself